dashboard/src-tauri/src/main.rs

```diff
@@ -50,12 +51,33 @@ async fn get_owner_jwt(state: tauri::State<'_, AppState>) -> Result<JwtToken, ()
 
 #[tokio::main]
 async fn main() {
-    let (core_fut, app_state, _guard, shutdown_tx) = lodestone_core::run(lodestone_core::Args {
+    let run_result = lodestone_core::run(lodestone_core::Args {
         is_cli: false,
         is_desktop: true,
         lodestone_path: None,
     })
     .await;
+
+    let (core_fut, app_state, _guard, shutdown_tx);
+    match run_result {
+        Ok((fut, state, guard, tx)) => {
+            core_fut = fut;
+            app_state = state;
+            _guard = guard;
+            shutdown_tx = tx;
+        }
+        Err(_) => {
```

Comment:
Might be good if we display the error message in the notification body
Ideally we can make it pretty

Reply:
added. wasn't sure if that's relevant information for the end user so i reserved from adding it before